website/blog/2021-04-12-funding-update.md

```diff
@@ -0,0 +1,70 @@
+
+We strongly believe that working in open source should be a viable and sustainable career path. We should be bringing everyone *up* to the same level, not *down*. However, we need to face the fact that this would mean draining [our current balance](https://opencollective.com/babel#category-BUDGET) in just a few months.
+
+Everyone taking smaller and smaller amounts ends up creating a false sense of sustainability. It lowers how we value our own work, and what sponsors perceive to be sustainable amount of funding.
+
+The reality is that, without more funding, we risk not being able to maintain the high standard of support and development that Babel and its users deserve. We risk losing more team members, who deserve to make a decent living in line with their skill level. The open source ecosystem also risks under-supporting a critical and widely used piece of technology.
+
+## Our Ask
+
+For now, Nicolò, Henry, and Jùnliàng will all be paid a temporary rate of \$6,000 per month. This doesn't solve the problem, but it gives us time while increasing the initial temporary \$2,000 USD salaries. The burn rate will still be more than the donations we currently receive each month, but our current balance allows us to sustain this amount until the end of 2021. 
+
+We'll do our best to balance all the priorities, while at reduced capacity due to being able to fund less of people's time and needing to spend some of that time finding new sponsors. Babel is not a company, a product, or service. And we have a small team, so we are the same people working on engineering and on funding. But it's a compromise we don't think we should have to make, considering the huge value that Babel delivers.
+
+To fully fund the currently paid maintainers (but we would like to expand who is being paid, to include the whole core team), we need at least $333,000 per year, which is 2x what we're currently bringing in. Considering the amounts of money in the tech sector, this seems tiny in comparison—but it would make a big difference to Babel. Our story and stories like it make a big difference to the health and sustainability of the open source ecosystem overall.
+
+**So, our ask is to help fund our work**, via [Open Collective](https://opencollective.com/babel) and [GitHub Sponsors](https://github.com/sponsors/babel). Though contributions from individuals do matter (and we deeply appreciate them), we are really looking for more companies to step up and become [corporate sponsors](https://opencollective.com/babel/contribute), alongside some of our current sponsors like AMP, Airbnb, Salesforce, and GitPod. If it would be better for your company to sustain us in other ways, we are also open to hearing any ideas. Reach out to us directly or by email at team@babeljs.io.
```

Comment:
```suggestion
**So, our ask is to help fund our work**, via [Open Collective](https://opencollective.com/babel) and [GitHub Sponsors](https://github.com/sponsors/babel). Though individual contributions do matter (and we deeply appreciate them), we are really looking for more companies to step up and become [corporate sponsors](https://opencollective.com/babel/contribute), alongside our current sponsors like AMP, Airbnb, Salesforce, and GitPod, etc. If it would be better for your company to sustain us in other ways, we are also open to hearing any ideas. Reach out to us directly or by email at team@babeljs.io.
```
Shorten a bit.